examples/examples_test.go

```diff
@@ -66,6 +66,26 @@ func TestExamples(t *testing.T) {
 				assert.Equal(t, body, "A File from Blob Storage")
 			}),
 		}),
+		base.With(integration.ProgramTestOptions{
+			Dir: path.Join(cwd, "msi-renamed-to-authorization"),
+			EditDirs: []integration.EditDir{
+				{
+					Dir:      "step2",
+					Additive: true,
+					ExpectNoChanges: true,
+				},
+			},
+		}),
+		base.With(integration.ProgramTestOptions{
+            Dir: path.Join(cwd, "servicebus-migration-test"),
```

Comment:
The indentation looks off? (tabs/spaces?)

Reply:
It would fail go-lint if it was an issue :)

Reply:
This is the beauty of go